libs/dh/dev-examples/feature-processes/src/details/details.component.ts

```diff
@@ -0,0 +1,193 @@
+            [label]="t('details.startedBy')"
+            [value]="process?.createdBy?.displayName | dhEmDashFallback"
+          />
+          <watt-description-list-item
+            [label]="t('details.executionTime')"
+            [value]="startedAtOrScheduledAt() | wattDate: 'long' | dhEmDashFallback"
+          />
+
+          <watt-description-list-item [label]="t('details.processId')" [value]="id()" />
+
+          <watt-description-list-item
+            [label]="t('details.period')"
+            [value]="process?.period | wattDate: 'short' | dhEmDashFallback"
+          />
+
+          @if (calculation) {
+            <watt-description-list-item
+              [label]="t('details.executionType')"
+              [value]="'shared.executionTypes.' + calculation.executionType | transloco"
+            />
+          }
+
+          @if (energyTimeSeriesRequest) {
+            <watt-description-list-item
+              [label]="t('details.meteringPointType')"
+              [value]="t('meteringPointType.' + energyTimeSeriesRequest.meteringPointType)"
+            />
+          }
+
+          @if (wholesaleRequest) {
+            <watt-description-list-item
+              [label]="t('details.priceType')"
+              [value]="t('priceType.' + wholesaleRequest.priceType)"
+            />
+          }
+        </watt-description-list>
+      </watt-drawer-heading>
+      <watt-drawer-content>
+        <dh-result [hasError]="hasError()" [loading]="loading()">
+          <vater-stack direction="row" offset="l" fill="horizontal">
+            <vater-flex fill="horizontal" gap="l" offset="l">
+              @if (process) {
+                <watt-progress-tracker>
+                  @for (step of process.steps; track step; let i = $index) {
+                    <watt-progress-tracker-step
+                      [label]="'shared.steps.' + i + '.pending' | transloco"
+                      [status]="step.state"
+                      [current]="step.isCurrent"
+                    >
+                      {{ 'shared.steps.' + i + '.succeeded' | transloco }}
+                    </watt-progress-tracker-step>
+                  }
+                </watt-progress-tracker>
+              }
+              @if (calculation) {
+                <vater-flex scrollable fill="vertical" grow="0">
+                  <dh-calculation-details-grid-areas [gridAreas]="calculation.gridAreas" />
+                </vater-flex>
+              }
+            </vater-flex>
+          </vater-stack>
+        </dh-result>
+      </watt-drawer-content>
+    </watt-drawer>
+  `,
+})
+export class DhProcessDetailsComponent {
+  private processQuery = lazyQuery(GetProcessByIdDocument);
+
+  navigation = inject(DhNavigationService);
+  loading = this.processQuery.loading;
+  hasError = this.processQuery.hasError;
+  result = computed(() => this.processQuery.data()?.processById);
+  startedAtOrScheduledAt = computed(() => this.result()?.scheduledAt);
```

Comment:
suggestion: now the naming implies that the value is either `startedAt` or `scheduledAt`, but it is always `scheduledAt`. I would add `startedAt` so it works like in calculations.
```suggestion
  startedAtOrScheduledAt = computed(() => this.result()?.startedAt ?? this.result()?.scheduledAt);
```

Reply:
Ofc, my bad :-)